### PR TITLE
CNF-14785: Validate new spoke CR to be managed by the ACM PG

### PR DESCRIPTION
**Pull request by irinamihai**

### CODE REVIEW

File: docs/cluster_configuration.md [hunk 1]
```diff
@@ -257,3 +257,63 @@ For updating a manifest in an existing ACM PolicyGenerator, the following steps
       policyNamespace: ztp-clustertemplate-a-v4-16
       remediationAction: enforce
     ```
+
+### Adding a new manifest to an existing ACM PolicyGenerator
+
+This usecase is identical to the [previous one](#updates-to-an-existing-acm-policygenerator-manifest), with the following distinctions:
+* If the new manifest does not have a corresponding `source-cr` file, the CSP should add a new yaml file to the `custom-crs` directory.
```
Comment (edcdavid) [inline docs/cluster_configuration.md:264]: what does CSP mean?

File: docs/cluster_configuration.md [hunk 2]
```diff
@@ -257,3 +257,63 @@ For updating a manifest in an existing ACM PolicyGenerator, the following steps
       policyNamespace: ztp-clustertemplate-a-v4-16
```
Comment (Missxiaoguo) [inline docs/cluster_configuration.md:257]: Not related to this PR! Should these namespaces be changed to `ztp-sno-ran-du-v4-16` which match the PG namespace?

File: docs/cluster_configuration.md (hunk 1 above)
Comment (irinamihai) [reply docs/cluster_configuration.md:264]: Cloud Service Provider

File: docs/cluster_configuration.md (hunk 2 above)
Comment (irinamihai) [reply docs/cluster_configuration.md:257]: Yes, good catch. I'll update all of them in a future PR. Thank you.

**Pull request merged**